focuspoints.lrdevplugin/focus_points/pentax/pentax k-50.txt

```diff
@@ -2,8 +2,6 @@
 -- Submitted by: roguephysicist
 -- 16.3 MP, 4928x3264
 -- AF System: 11 points, 9 cross-type
--- selected points: AFPointSelected
--- in-focus points: AFPointsInFocus
```

Comment:
sounds good. i can make this a future feature work (on myself). I think it'd be cool.